### PR TITLE
Cart management page

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**
Added Cart Management page 

Live Link :-  https://happy-kalam-c6d6de.netlify.app/cartmanagement

### CODE REVIEW

File: cartManagement.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      <section
+        class="cart_container d-flex productlisting_main_content_container"
+      >
+        <!-- Product items container -->
+        <div class="cart_products_container d-flex">
+          <div
+            class="card-basic Card-horizontalImage-textContainer cart_card_resize p-relative"
+          >
+            <div class="Card-horizontalImage-text">
+              <img
```
Comment (Maianki) [inline cartManagement.html:80]: add alt attribute on image for Accessibility purposes.

File: cartManagement.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+            </div>
+          </div>
+
+          <div
+            class="card-basic Card-horizontalImage-textContainer cart_card_resize p-relative"
+          >
+            <div class="Card-horizontalImage-text">
+              <img
+                class="card-img Card-horizontalImage-text-img Card-horizontalImage-text-img-Ecommerce cart_card_imgresize"
+                src="https://pyxis.nymag.com/v1/imgs/4a7/29b/4ef2f518834dd81d8ceb374caa54fd5171-22-plant-gifts.rsquare.w700.jpg"
```
Comment (Maianki) [inline cartManagement.html:123]: Here also use alt attribute.

File: cartManagement.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+            </div>
+          </div>
+
+          <div
+            class="card-basic Card-horizontalImage-textContainer cart_card_resize p-relative"
+          >
+            <div class="Card-horizontalImage-text">
+              <img
+                class="card-img Card-horizontalImage-text-img Card-horizontalImage-text-img-Ecommerce cart_card_imgresize"
+                src="https://cdn.shopify.com/s/files/1/0317/0687/3992/products/232-large_default.jpg?v=1580435931"
```
Comment (Maianki) [inline cartManagement.html:164]: Use alt attribute.

**Review by Maianki — CHANGES_REQUESTED** [review]
Suggested a few changes. Everything else looks good.

File: cartManagement.html (hunk 1 above)
Comment (abhisheksahani-nova) [reply cartManagement.html:80]: Done

File: cartManagement.html (hunk 2 above)
Comment (abhisheksahani-nova) [reply cartManagement.html:123]: Added

File: cartManagement.html (hunk 3 above)
Comment (abhisheksahani-nova) [reply cartManagement.html:164]: Added

File: styles/cartManagement.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  height: fit-content;
+}
+
+.cart_card_resize {
+  width: 39rem;
+}
+
+.cart_card_content {
+  margin-top: 6px;
+}
```
Comment (anik31) [inline styles/cartManagement.css]: Use rem here since you are using rem everywhere, even for small sizes.

Comment (abhisheksahani-nova) [reply styles/cartManagement.css]: I know rem can be used , but I am following what Vikram said , using pixels for value less than 10

Comment (anik31) [reply styles/cartManagement.css]: Yes I know that. But just below this code you have used 0.1rem for padding, isn't it less than 10px ?

**Review by Maianki — APPROVED** [review]
Looks good to merge.

File: styles/cartManagement.css (hunk 4 above)
Comment (abhisheksahani-nova) [reply styles/cartManagement.css]: okay , I will change

Comment (abhisheksahani-nova) [reply styles/cartManagement.css]: Did the changes

**Review by anik31 — APPROVED** [review]
Nice work. CSS class naming is good.

**Pull request merged**